Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -120,19 +114,15 @@ private String overrideNullQuery(String query) {
      *            of searched document
      * @return JSONObject
      */
-    public JSONObject findDocument(Integer id) throws DataException {
+    public JsonObject findDocument(Integer id) throws DataException {
         SearchRestClient restClient = initiateRestClient();
-        JSONParser parser = new JSONParser();
 
         String response = restClient.getDocument(id);
         if (!response.equals("")) {
-            try {
-                return (JSONObject) parser.parse(response);
-            } catch (ParseException e) {
-                throw new DataException(e);
-            }
+            JsonReader jsonReader = Json.createReader(new StringReader(response));
```

Comment:
Same here. `JsonReader` is closeable so using a try-with-resources statement for closing the object is maybe a good idea.